scripts/setup_iso.py

```diff
@@ -153,6 +153,9 @@ def determine_os_ver(srcdir, iso_info):
             print 'attempting LiveCD netboot'
             osname = 'LiveCD'
             osver = vid
+            for i in src_dir_list:
+                if 'PHOTON' in i:
+                    osname = 'PHOTON'
```

Comment:
It's better TO
Change
`elif 'isolinux' in src_dir_list:`
to
`elif 'LiveOS' in src_dir_list and 'isolinux' in src_dir_list:`

And change
```
else:
    for i in src_dir_list:
        if "PHOTON" in i:
            osname = "PHOTON"
            if vid == '':
                assert ValueError, "The volume ID is not get correctly"
            osver = vid.lstrip('PHOTON_')
```
to
```
elif 'PHOTON_VMWARE_CD' in src_dir_list:
    osname = "PHOTON"
    if vid == '':
        assert ValueError, "The volume ID is not get correctly"
    osver = vid.lstrip('PHOTON_')
```